00-open-a-shapefile.Rmd

```diff
@@ -72,16 +72,15 @@ Crop and extract raster values in R</a>
 NOTE: The data used in this tutorial were collected at Harvard Forest which is
 a the National Ecological Observatory Network field site 
 <a href="http://www.neoninc.org/science-design/field-sites/harvard-forest" target="_blank">
-More about the NEON Harvard Forest field site</a>. These data are proxy data for what will be
-available for 30 years from the NEON flux tower 
+More about the NEON Harvard Forest field site</a>. These data are a preview of the 30 year time-series that will be collected from the NEON flux tower and made available
```

Comment:
Ok, I will change it back. It was just that I thought some people would be expecting to work with a 30 year dataset after reading.